reactor-core/src/test/java/reactor/util/context/ContextNTest.java

```diff
@@ -37,6 +38,74 @@ public void initContext() {
 			4, "D", 5, "E", 6, "F");
 	}
 
+	@Test
+	public void constructFromPairsRejectsNulls() {
+		assertThatNullPointerException().isThrownBy(() -> new ContextN(null, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6));
```

Comment:
I'm a bit afraid of having to change 12 lines if we for instance change the exception type :)
WDYT about something like:
```java
final int argsNum = 12;
for (int arg = 0; arg < argsNum; arg++) {
    Object[] args = IntStream.range(0, argsNum).boxed().toArray();
    args[arg] = null;
    assertThatNullPointerException().isThrownBy(() -> new ContextN(args));
}
```

Reply:
this `ContextN` constructor is not taking an array

Reply:
Ooops.... Sorry :)